runtime/commonMain/src/kotlinx/serialization/SerialFormat.kt

```diff
@@ -29,6 +29,7 @@ interface BinaryFormat: SerialFormat {
     fun <T> load(deserializer: DeserializationStrategy<T>, bytes: ByteArray): T
 }
 
+// TODO migrate
 fun <T> BinaryFormat.dumps(serializer: SerializationStrategy<T>, obj: T): String =
     HexConverter.printHexBinary(dump(serializer, obj), lowerCase = true)
```

Comment:
btw, we should eventually make a decision on this `HexConverter` class, because it is public now. And we don't plan to provide any analogs in `kotlinx-io`, do we?

Reply:
We do. `HexConverter` should be deprecated and eventually hidden from `kotlinx.serialization`